tests/unit/test_in_memory_cache.py

```diff
@@ -54,7 +54,9 @@ def test_get_nonexistent_user(cache):
 def test_get_improper_user_id(cache):
     """Test how improper user ID is handled."""
     with pytest.raises(ValueError):
-        assert cache.get("foo/bar", conversation_id) is None
+        assert cache.get(":", conversation_id) is None
```

Comment:
Why you are comparing that to None instead of matching the error msg?

Reply:
updated @onmete